Modules/Core/Common/test/itkImageBaseGTest.cxx

```diff
@@ -119,6 +119,65 @@ Check_New_ImageBase()
   EXPECT_TRUE(imageBase->GetInverseDirection().GetVnlMatrix().is_identity());
 }
 
+template <unsigned int VImageDimension>
+void
+CheckInvalidSpacingExceptions()
+{
+  using SpacingValueType = typename itk::ImageBase<VImageDimension>::SpacingValueType;
+  using SpacingType = typename itk::ImageBase<VImageDimension>::SpacingType;
+  using DirectionType = typename itk::ImageBase<VImageDimension>::DirectionType;
+
+  const auto imageBase = itk::ImageBase<VImageDimension>::New();
+
+  // Test exceptions
+  SpacingValueType spacingValue = -1.0;
+  SpacingType      spacing;
```

Comment:
Just a little suggestion: instead of reusing the very same local `spacing` variable, I would use three `const` spacings, to further improve code readability:

    const SpacingType initialSpacing = imageBase->GetSpacing();
    const SpacingType negativeSpacing(-1.0);
    const SpacingType zeroSpacing(0.0);

Reply:
Thanks for the review. Reads definitely more cleanly. Force pushed in  8c06b7a.